lib/widgets/carousel.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:carousel_slider/carousel_slider.dart';
+import 'package:flutter/material.dart';
+
+final List<String> imgList = [
+  'https://sochem.in/static/media/slide-home-3.d001fcc5.jpg',
+  'https://osmoze.in/assets/images/gallery/2.png',
+  'https://iitbhu.ac.in/sites/default/files/styles/iitbhu_slides/public/slides/che/slide_che_02.jpg?itok=R1Cmzwfi',
+  'https://osmoze.in/assets/images/gallery/4.JPG',
+  'https://images.unsplash.com/photo-1567427018141-0584cfcbf1b8?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2070&q=80',
+  'https://images.unsplash.com/photo-1565354785692-9e7523e5a87b?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1035&q=80',
+];
+final List<Widget> slidingImages = imgList
+    .map((item) => Container(
+          child: Container(
+            decoration: BoxDecoration(
+              boxShadow: [
+                BoxShadow(
+                  color: Colors.black38,
+                  offset: const Offset(
+                    1.0,
+                    1.0,
+                  ), //Offset
+                  blurRadius: 4.0,
+                ),
+              ],
+            ),
+            margin: EdgeInsets.all(5.0),
+            child: ClipRRect(
+                borderRadius: BorderRadius.all(Radius.circular(5.0)),
+                child: Stack(
+                  children: <Widget>[
+                    Image.network(
+                      item,
+                      fit: BoxFit.cover,
+                      width: 1000.0,
+                      alignment: Alignment.center,
+                    ),
+                    Positioned(
+                      bottom: 0.0,
+                      left: 0.0,
+                      right: 0.0,
+                      child: Container(
+                        decoration: BoxDecoration(
+                          gradient: LinearGradient(
+                            colors: [
+                              Color.fromARGB(200, 0, 0, 0),
+                              Color.fromARGB(0, 0, 0, 0)
+                            ],
+                            begin: Alignment.bottomCenter,
+                            end: Alignment.topCenter,
+                          ),
+                        ),
+                        padding: EdgeInsets.symmetric(
+                            vertical: 10.0, horizontal: 20.0),
+                        child: Text(
+                          'No. ${imgList.indexOf(item) + 1} image',
+                          style: TextStyle(
+                            color: Colors.white,
```

Comment:
Also this would be removed later on because it is not needed actually for production app right?

Reply:
Yes, I included these for considering adding a little highlight or detail of the picture being shown, be it of any event or occasion or anything, a small text notifying what is that picture about. This can also be used with the required API but if not required can be removed.